src/ts/util.ts

```diff
@@ -53,6 +53,16 @@ const isSnowplow = (request: Request): boolean => {
   return false;
 };
 
+type ScenarioTrigger = {
+  description: string
+  appIds?: string[],
+  url?: string,
+  variantUrls?: {
```

Comment:
I think this is `Record<string, string>` from the schema, but not important since we don't actually use it.